src/AddTodoForm.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const AddTodoForm = () => {
+  return (
+    <>
```

Comment:
This react fragment is unnecessary since you're only returning one component or HTML tag (<form> nested elements </form>). However, I don't think this causes any difference in the way the page renders. If returning multiple sibling components (existing side by side rather than nested within one parent component) then a react Fragment (or empty tag) is necessary to enclose those sibling components within as the return value.

Here's a good discussion on the difference between <React.Fragment></React.Fragment> and <></>:

https://medium.com/fasal-engineering/what-are-react-fragments-or-the-react-empty-tags-190253582905